svg-convert.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+
+source_file="$1"
+dest_size="$2"
+dest_file="$3"
+
+exists () {
+	type "$1" >/dev/null 2>/dev/null
+}
+
+divide () {
+	echo - | awk "{ print $1 / $2 }"
+}
+
+if exists lasem-render; then
+	CONVERT=lasem-render
+elif exists lasem-render-0.4; then
+	CONVERT=lasem-render-0.4
+elif exists rsvg-convert; then
+	CONVERT=rsvg-convert
+elif exists gm; then
+	CONVERT=gm
+elif exists convert; then
+	CONVERT=convert
+else
+	echo "No supported SVG converter found! (Tried: lasem, rsvg, imagemagick)" >&2
+	exit 1
+fi
+
+case $CONVERT in
+	lasem-render*)
+		source_size=$(lasem-render-0.4 --debug render "${source_file}" -o /dev/null | awk '/width/ { print $3 }')
```

Comment:
`lasem-render-0.4` should probably be `$CONVERT` or there will be error if there is only `lasem-render` executable.